src/NuGetForUnity/Editor/NugetPreferences.cs

```diff
@@ -20,6 +20,11 @@ public class NugetPreferences : SettingsProvider
         /// </summary>
         private static Vector2 scrollPosition;
 
+        /// <summary>
+        ///     Indicates if the warning for packages.config file path should be shown in case it is outside of Assets folder.
+        /// </summary>
+        private static bool shouldShowPackagesConfigPathWarning;
```

Comment:
I don't see a reason for this and scrollPosition fields to be static. See if it works when you change them not to be static.